randovania/games/samus_returns/json_data/Area 6.txt

```diff
@@ -555,7 +555,22 @@ Extra - asset_id: collision_camera_043
   > Door to Diggernaut Shaft
       After Area 6 - Diggernaut and Lay Power Bomb
   > Before Diggernaut
-      Bomb and Phase Drift and Plasma Beam and Space Jump and Spring Ball and Shoot Beam Burst and Use Spider Ball
+      All of the following:
+          Lay Bomb
+          Any of the following:
+              # Energy Requirements
+              Combat (Hypermode) or Normal Damage ≥ 499
+              Combat (Beginner) and Normal Damage ≥ 399
+              Combat (Intermediate) and Normal Damage ≥ 299
+              Combat (Advanced) and Normal Damage ≥ 199
+          Any of the following:
+              # Dealing Damage
+              Plasma Beam and Shoot Beam Burst
+              Missile ≥ 24 and Combat (Expert)
```

Comment:
1. why is beam burst required?
2. does diggernaut only take damage if you have plasma?
3. whats with the 24 missile req?

Reply:
1/2. Best output. I honestly don't remember if you need beam Burst to damage it
3. Low% strat